templates/docs/examples/patterns/slider/slider-input.html

```diff
@@ -1,16 +1,40 @@
-{% extends "_layouts/examples.html" %}
-{% block title %}Slider / Input{% endblock %}
-
-{% block standalone_css %}patterns_slider{% endblock %}
-
-{% block content %}
-<div class="p-slider__wrapper">
-  <input class="p-slider" type="range" min="0" max="100" value="50" step="1" id="slider3">
-  <input class="p-slider__input" type="text" maxlength="3" id="slider3-input">
+{% extends "_layouts/examples.html" %} {% block title %}Slider / Input{% endblock %} {% block standalone_css %}patterns_slider{% endblock %} {% block content %} <div class="p-slider__wrapper">
+  <input class="p-slider"
+    type="range"
+    aria-value-min="0"
```

Comment:
Is this needed when there is `min` defined?
Seems like a lot of duplication to keep all values in 2 places...

Reply:
I couldn't find explicit informaitno that it is not needed, and saw an example at mdn that hade both (link in the attached issue) so figured it is the safer approach...